pages/about.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="/styles/about_style.css" rel="stylesheet">
 </head>
 <body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="/pages/index.html">HOME</a></li>
+        <li><a href="/pages/about.html">ABOUT</a></li>
+        <li><a href="/pages/portfolio.html">PORTFOLIO</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="container">
+    <h1>Tinghui Jiang</h1>
+    <h2>Software Developer</h2>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)